app/validators/questionnaire_schema.py

```diff
@@ -360,6 +360,14 @@ def get_all_answer_ids(self, block_id):
             for answer in self.get_answers_from_question(question)
         }
 
+    def get_all_dynamic_answer_ids(self, block_id):
```

Comment:
We probably need to sync our PRs and use the same method in both so there is no duplication of methods, I will use your method as it has better name and it's more universal.

Reply:
Ahh good point - thanks Pete 👍